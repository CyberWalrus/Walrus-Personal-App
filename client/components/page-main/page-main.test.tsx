import * as Enzyme from "enzyme";
import { shallow } from "enzyme";
import * as Adapter from "enzyme-adapter-react-16";
import toJson, { Json } from "enzyme-to-json";
import * as React from "react";
import PageMain from "./page-main";

Enzyme.configure({adapter: new Adapter()});
it(`PageMain correctly render`, (): void => {
  const tree: Json = toJson(shallow(<PageMain />));
  expect(tree).toMatchSnapshot();
});
