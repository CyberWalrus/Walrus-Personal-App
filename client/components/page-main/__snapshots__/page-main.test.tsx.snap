// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageMain correctly render 1`] = `
<div>
  Hello Walrus
</div>
`;
